### PR TITLE
fix torch.cumsum docs

### PR DESCRIPTION
**Pull request by jeejeelee**

### CODE REVIEW

**Review by zou3519 — CHANGES_REQUESTED** [review]
This is expected due to floating-point differences: the printing of a does not actually show all the digits. A better example would probably be to construct some sequence of numbers (ints to make things easier to visualize), and then call cumsum on it.

**Comment by jeejeelee** [comment]
@zou3519  Thanks for your review. I am still a rookie when it comes to PR, Does it mean that I should modify torch.cumsum docs in the following way?
```doc

    >>> a = torch.randint(1, 20, (10,))
    >>> a
    tensor([ 5,  8, 12, 15, 11,  6, 11, 16, 15,  8])
    >>> torch.cumsum(a, dim=0)
    tensor([  5,  13,  25,  40,  51,  57,  68,  84,  99, 107])
```

**Comment by zou3519** [comment]
> @zou3519 Thanks for your review. I am still a rookie when it comes to PR, Does it mean that I should modify torch.cumsum docs in the following way or close this PR?
> 
> ```
> 
>     >>> a = torch.randint(1, 20, (10,))
>     >>> a
>     tensor([ 5,  8, 12, 15, 11,  6, 11, 16, 15,  8])
>     >>> torch.cumsum(a, dim=0)
>     tensor([  5,  13,  25,  40,  51,  57,  68,  84,  99, 107])
> ```

That sounds reasonable to me

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -3413,13 +3413,11 @@ def merge_dicts(*dicts):
...
-    >>> a = torch.randn(10)
+    >>> a = torch.randint(1, 20, (10,))
     >>> a
-    tensor([-0.8286, -0.4890,  0.5155,  0.8443,  0.1865, -0.1752, -2.0595,
-             0.1850, -1.1571, -0.4243])
+    tensor([13,  7,  3, 10, 13,  3, 15, 10,  9, 10])
     >>> torch.cumsum(a, dim=0)
-    tensor([-0.8286, -1.3176, -0.8021,  0.0422,  0.2287,  0.0535, -2.0060,
-            -1.8210, -2.9781, -3.4024])
+    ttensor([13, 20, 23, 33, 46, 49, 64, 74, 83, 93])
```
Comment (zou3519) [inline torch/_torch_docs.py:3420]: tensor with one t

Comment (jeejeelee) [reply torch/_torch_docs.py]: I have already corrected this error.

**Review by zou3519 — APPROVED** [review]

**Comment by jeejeelee** [comment]
@zou3519 ,some checks were not successful ,what should Ido ?

**Pull request closed without merging**